nurseconnect/forms.py

```diff
@@ -0,0 +1,67 @@
+from django import forms
+from django.forms import Form
+from django.utils.translation import ugettext_lazy as _
+from molo.profiles.forms import RegistrationForm, EditProfileForm
+from nurseconnect import constants
+
+
+class NurseConnectRegistrationForm(RegistrationForm):
+    first_name = forms.CharField(
+        label=_("Name"),
+        widget=forms.TextInput(
+            attrs=dict(
+                required=True,
+                max_lenght=30
+            )
+        ),
+    )
+
+    last_name = forms.CharField(
+        label=_("Surname"),
+        widget=forms.TextInput(
+            attrs=dict(
+                required=True,
+                max_lenght=30
+            )
+        ),
+    )
+
+    gender = forms.ChoiceField(
```

Comment:
Don't set the choices if they are already set on the model field.  Set the model the form needs to use, unless RegistrationForm is not a ModelForm, in which case you will need to ensure the model is saved.